packages/fast-components-class-name-contracts/src/index.ts

```diff
@@ -1,2 +1,3 @@
+export * from "./divider";

```

Comment:
should be alphabetical

Reply:
@chrisdholt could you file a new issue to enforce this in fast-tslint-rules, I think @nicholasrice makes a good point.

Reply:
Still not sure it's a nit, although I agree that tslint should catch this. More than happy to discuss various options with the team. We should have some kind of order though aside from random import/exportss in my opinion.